### ISSUE TITLE
Add PR check/ automated report generattion to GH actions

### ISSUE THREAD

**Issue by marianntapfer** [TLU-Human-computer-interaction/masterThesis#30]
Steps: 

- [ ] Report should be automatically generated by github actions in every PR
- [ ] Report should be published in storybook
- [ ] Report should have a date when it was created
- [ ] Report should give developer warning if the number of accessibility issues increased

Issues that need to be solved for this
- [ ] generating report locally takes about 3.50 minutes. It might take even longer in GH actions
- [ ] Can we only check the stories that where changed or that use a component that was changed

**Comment by marianntapfer** [comment]
The CLI tool for generating the report does not work in storybook 7 and storybook 7 has a much better way to run accessibility tests on every PR. I see no point in generating the report. running tests is much better.
